src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/Views/MicrosoftEntraIDSettings.Edit.cshtml

```diff
@@ -26,7 +26,7 @@
     <label asp-for="TenantId" class="form-label">@T["TenantId"]</label>
     <input asp-for="TenantId" class="form-control" />
     <span asp-validation-for="TenantId"></span>
-    <span class="hint">@T["The Tenant (Directory) ID found on the Microsoft Entra ID App registration parameters."]</span>
+    <span class="hint">@T["The Tenant (Directory) ID found on the Microsoft Entra ID App registration parameters. For multitenant App registrations, use 'common' or 'organizations'."]</span>
```

Comment:
```suggestion
    <span class="hint">@T["The Tenant (Directory) ID found on the Microsoft Entra ID App registration parameters. For multi-tenant App registrations, use 'common' or 'organizations'."]</span>
```

Reply:
Microsoft mostly uses "multitenant" in their documentation and that was used in the Orchard doc as well. Are you sure to change that?

Reply:
Sure, I go for what you like most 😊